rafs/src/metadata/layout/v6.rs

```diff
@@ -174,22 +181,41 @@ impl RafsV6SuperBlock {
             )));
         }
 
-        if self.s_inos == 0 {
+        if u64::from_le(self.s_inos) == 0 {
             return Err(einval!("invalid inode number in Rafsv6 superblock"));
         }
 
         if self.s_extslots != 0 {
             return Err(einval!("invalid extended slots in Rafsv6 superblock"));
         }
 
-        if self.s_blocks == 0 {
+        if u32::from_le(self.s_blocks) == 0 {
             return Err(einval!("invalid blocks in Rafsv6 superblock"));
         }
 
-        if self.s_u != 0 {
+        if u16::from_le(self.s_u) != 0 {
             return Err(einval!("invalid union field in Rafsv6 superblock"));
         }
 
+        if u32::from_le(self.s_xattr_blkaddr) != 0 {
```

Comment:
this is the start blkaddr of shared xattrs

Reply:
I think it's designed for that, but not used yet.